### DISCUSSION TITLE
"E: Unable to correct problems, you have held broken packages."

### DISCUSSION THREAD

**Discussion by water-logger** [termux/termux-app#3393] [Q&A] [answered]
Hi, I'm trying to install the Termux Desktop and I get this error: 
![The error](https://github.com/termux/termux-app/assets/101909986/e9254f24-b07f-4575-9e96-fed8f7f0840d) I'm not do good at doing things with packages so how would I fix this?

**Comment by sylirre** [comment, marked as answer]
I suggest to begin with `pkg upgrade` and then install needed packages such as errored `ncurses-utils`.
